src/main/java/org/xerial/snappy/OSInfo.java

```diff
@@ -215,6 +216,10 @@ public static String getArchName() {
             return "android-arm";
         }
 
+        if (isMusl()) {
+            return "x86_64-musl";
```

Comment:
It seems musl is available for arm64 (aarhc64) as well. Always resolving to x86_64 is not correct. Should we rename it to isX64Musl() to avoid confusion?